snmptrap.php

```diff
@@ -21,22 +23,32 @@
 $init_modules = array();
 require __DIR__ . '/includes/init.php';
 
-$entry = explode(',', $argv[1]);
+// Creates an array with trap info
+while ($f = fgets(STDIN)){
```

Comment:
Why the switch away from `$argv[1]`?

Reply:
Dont ask me why but $argv[] only contains the script itself so I used other approach to parse info

Reply:
Are you sure that wasn't just `$argv[0]` that contained the script?

Reply:
Well... I'm almost sure. I've tried to dump $argv with `var_export($argv, true)` to know what were in the array and only the script was dumped.

With my new implementation, dumping $entry with `var_export($entry, true)` after the while dumped everything recieved in the trap.

As you know, Im not really a developer so I may have missed something that makes $argv to only dump the script.